src/setup/unattended-install/config-tool-ref.md

```diff
@@ -24,33 +24,46 @@ The Configuration Tool command line returns non-zero values when an error occurs
 ## Syntax
 
 ```
-ConfigurationTool.com {/SetupInstall <platform_db_admin_username> <platform_db_admin_password> <logging_db_admin_username> <logging_db_admin_password> [/SetPlatformServerAdminPassword <platform_server_admin_password>] | /UpgradeInstall [<admin_password>] [/SetPlatformServerAdminPassword <platform_server_admin_password>]} [/RebuildSession <session_db_admin_username> <session_db_admin_password>] [/SCInstall] [/CreateUpdateCacheInvalidationService]
+ConfigurationTool.com 
+    | /SetupInstall <platform_db_admin_username> <platform_db_admin_password> <logging_db_admin_username> <logging_db_admin_password> [/SetPlatformServerAdminPassword <platform_server_admin_password>] 
+    | /UpgradeInstall [<integrated_auth_admin_password>] [/SetPlatformServerAdminPassword <platform_server_admin_password>] 
+    | /RebuildSession <session_db_admin_username> <session_db_admin_password> 
+    | /SCInstall 
+    | /UpgradeSystemComponents
+    | /UpgradePublishedApplications
     | /GenerateTemplates
     | /ClearInternalNetwork
-    | /UploadLicense <license_file>
+    | /UploadLicense <license_file> <platform_server_admin_user> <platform_server_admin_password>
     | /RegenerateSettingsKey
     | /GetSerial
+    | /SetPlatformServerAdminPassword <platform_server_admin_password>
     | /GetDeploymentZones
-    | /ModifyDeploymentZone <configuration_name> <zone_address> [<enable_https>]
+    | /ModifyDeploymentZone <deployment_zone_name> <deployment_zone_address> [<enable_https>]
+    | /CreateUpgradeCacheInvalidationService
     | /EnableServerAPI
     | /DisableServerAPI
+    | /UpgradeEnvironment
 ```
 
 ## Parameters
 
-`/SetupInstall [<platform_db_admin_username> <platform_db_admin_password>] [<logging_db_admin_username> <logging_db_admin_password>]`
+`/SetupInstall [<platform_db_admin_username>] [<platform_db_admin_password>] [<logging_db_admin_username>] [<logging_db_admin_password>] [/SetPlatformServerAdminPassword platform_server_admin_password]`
```

Comment:
This updated syntax is different from the global syntax above:

At the top (syntax summary):

> `/SetupInstall <platform_db_admin_username> <platform_db_admin_password> <logging_db_admin_username> <logging_db_admin_password> [/SetPlatformServerAdminPassword <platform_server_admin_password>]`

vs. here:

> `/SetupInstall [<platform_db_admin_username>] [<platform_db_admin_password>] [<logging_db_admin_username>] [<logging_db_admin_password>] [/SetPlatformServerAdminPassword platform_server_admin_password]`

Can you supply the usernames without the passwords for platform and logging DBs?

Reply:
> 
> 
> This updated syntax is different from the global syntax above:
> 
> At the top (syntax summary):
> 
> > `/SetupInstall <platform_db_admin_username> <platform_db_admin_password> <logging_db_admin_username> <logging_db_admin_password> [/SetPlatformServerAdminPassword <platform_server_admin_password>]`
> 
> vs. here:
> 
> > `/SetupInstall [<platform_db_admin_username>] [<platform_db_admin_password>] [<logging_db_admin_username>] [<logging_db_admin_password>] [/SetPlatformServerAdminPassword platform_server_admin_password]`
> 
> Can you supply the usernames without the passwords for platform and logging DBs?

We can install without any arguments, but it does indeed not make much sense if we try to just use some of the arguments.